source/_patterns/04-components/button/button.twig

```diff
@@ -4,12 +4,12 @@
 ]|join(' ')|trim %}
 
 {% if is_button_tag %}
-  <button class="{{ classes }}"{% if is_disabled %} disabled{% endif %}>{{ text }}</button>
+  <button class="{{ classes }}"{% if is_disabled %} disabled{% endif %}{% if button_type %} type="{{button_type }}"{% endif %}>{{ text }}</button>
```

Comment:
Please add a space before the button_type variable where it's printed out.  Otherwise looks good!

Reply:
nm, i see an unrelated issue in these files so i'll address the variable name spacing when i fix the other issues.